core/server/web/middleware/validation/profile-image.js

```diff
@@ -0,0 +1,23 @@
+var common = require('../../../lib/common'),
+    imageLib = require('../../../lib/image');
+
+module.exports = function profileImage() {
+    // we checked for a valid image file, now we need to do validations for profile image
+    return function profileImageValidation(req, res, next) {
+        return imageLib.blogIcon.getIconDimensions(req.file.path).then(function (response) {
+            // save the image dimensions in new property for file
+            req.file.dimensions = response;
```

Comment:
Is `req.file.dimensions` used later on?

Reply:
Later in the file yes, to check for square dimensions, but not anywhere after. Do you think it makes sense to keep it in that case(was copied over from blog icon validation)?